### PR TITLE
#304 missing member poster

### LINKED ISSUE #304: integrate missing member poster on front end

### PR DESCRIPTION
**Pull request by pdvorchik**
This closes #304.

Please test the MoC cards (after a zip code search) as well as the missing member MoC cards.

### CODE REVIEW

File: scripts/views/repCardView.js [hunk 1]
```diff
@@ -30,6 +30,7 @@
   repCardHandler.renderRepresentativeCards = function(representativePromise, $parent, state) {
     $parent.empty(); // If they search for a new zipcode clear the old info
     representativePromise.then(function(representatives) {
+      console.log(representatives);
       var compiledTemplate = Handlebars.getTemplate('representativeCard');
```
Comment (meganrm) [inline scripts/views/repCardView.js]: can you remove this log?

Comment (pdvorchik) [reply scripts/views/repCardView.js]: Ooops!  Fixed.

Comment (meganrm) [reply scripts/views/repCardView.js]: it looks like it's still there

Comment (pdvorchik) [reply scripts/views/repCardView.js]: Gah!  Forgot to stage during my amend commit.

**Pull request merged**